src/foam/nanos/auth/CapabilityAuthService.js

```diff
@@ -127,13 +128,14 @@ foam.CLASS({
               NOT(HAS(UserCapabilityJunction.EXPIRY)),
               NOT(EQ(UserCapabilityJunction.STATUS, CapabilityJunctionStatus.EXPIRED))
           );
-          Predicate predicate = new CapabilityAuthServicePredicate(x, capabilityDAO, permission);
+          CapabilityAuthServicePredicate predicate = new CapabilityAuthServicePredicate(x, capabilityDAO, permission, null);
```

Comment:
couldn't find this constructor - was wondering what null is here.
<img width="1440" alt="Screen Shot 2022-01-26 at 8 41 37 AM" src="https://user-images.githubusercontent.com/10802216/151177137-4a746767-4c69-4062-9be6-63da37835199.png">
assuming there's another pr...